src/js/engagement_view/src/components/graphDisplay/GraphDisplay.tsx

```diff
@@ -39,36 +32,42 @@ const defaultClickedState = (): ClickedNodeState => {
     return null;
 };
 
+async function updateGraphAndSetState(
+    lensName: any,
+    state: any,
+    setState: any
+) {
+    if (lensName) {
+        await updateGraph(lensName, state as GraphState, setState); // state is safe cast, check that lens name is not null
+
+        console.log(
+            "setState",
+            await updateGraph(lensName, state as GraphState, setState)
+        );
```

Comment:
This updates the graph twice in a row. If you want to log the return value of updateGraph do `let merged = await updateGraph` and log merged.

Reply:
This was for testing, sorry, removing